### DISCUSSION TITLE
Customizable layer height management in Cura

### DISCUSSION THREAD

**Discussion by 5axes** [smartavionics/Cura#87] [Ideas]
Do you think that one day it will be possible to have a customizable layer height management in Cura ? And I don't talk about the experimental Adaptative Layer Height function.

**Comment by Sophist-UK** [comment]
Personally I don't have much idea what you mean when you talk about "customizable layer height management". Can you explain further?

**Reply by 5axes** [reply]
Possibility to define differents Layer Heights for one project.  As we can have in the other slice
